src/compiler/crystal/codegen/fun.cr

```diff
@@ -202,6 +202,17 @@ class Crystal::CodeGenVisitor
         end
       end
 
+      if @program.has_flag?("wasm32")
+        if target_def.is_a? External && (wasm_import_module = target_def.wasm_import_module)
```

Comment:
Just to avoid potential confusion as to what's the argument here.

```suggestion
        if target_def.is_a?(External) && (wasm_import_module = target_def.wasm_import_module)
```